packages/larva-patterns/modules/content-showcase/content-showcase.prototype.js

```diff
@@ -0,0 +1,23 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const c_lazy_image = clonedeep( require( '@penskemediacorp/larva-patterns/components/c-lazy-image/c-lazy-image.prototype' ) );
+const c_title = clonedeep( require( '@penskemediacorp/larva-patterns/components/c-title/c-title.prototype' ) );
+const c_dek = clonedeep( require( '@penskemediacorp/larva-patterns/components/c-dek/c-dek.prototype' ) );
+const c_link = clonedeep( require( '@penskemediacorp/larva-patterns/components/c-link/c-link.prototype' ) );
+
+c_lazy_image.c_lazy_image_crop_class = 'lrv-a-crop-1x1';
+c_lazy_image.modifier_class = 'u-width-100p lrv-u-margin-l-auto';
+
+c_link.c_link_classes = 'a-font-body-l lrv-a-floated-left lrv-a-hover-effect-text-decoration--underline lrv-u-margin-b-00'
```

Comment:
All classes should be lrv- namespaced when they are added to the larva packages - `lrv-a-font-body-l` here.

Reply:
@laras126 This has been fixed in the updated template